docs/src/_reference/command-line-interface.md

```diff
@@ -486,7 +486,7 @@ meltano init [project_name] [--no_usage_stats]
 
 #### Parameters
 
-- **project_name** - This determines the folder name for the project
```

Comment:
I think `project_name` should be renamed to `project_directory`, both here, in `src/meltano/core/project_init_service.py`, and wherever else the term is used.

Reply:
Aside from that, I think this is good to go. Thanks @ReubenFrankel!

Reply:
I agree - this did cross my mind but wondered if it was too much of a change initially.

Reply:
The name has always been misleading imo, since Meltano does not store or ever use this "project name" for anything other than creating the directory.

Reply:
https://github.com/meltano/meltano/pull/6814/commits/1c7a5ec2d9b5ab02671d1bae27c3f8d3b5b7bd18